docker/fv3net/Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN chmod +x /usr/local/bin/argo
 # install gcloud sdk
 RUN gcloud config set project vcm-ml
 
+# install git and git-lfs (needed to install torch_harmonics from GitHub)
```

Comment:
do we actually need git-lfs? I know you can clone the repo even without git-lfs installed (the "lfs" parts just don't get pulled, I think) although maybe there is some issue with pip installing from github?

Reply:
I see, thanks for checking.